### PR TITLE
docs(cubestore): update `README.md`

### PR DESCRIPTION
**Pull request by hassankhan**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

### CODE REVIEW

**Review by rpaik — APPROVED** [review]

File: rust/README.md [hunk 1]
```diff
@@ -1,36 +1,75 @@
...
+[Slack](https://slack.cube.dev) • [Twitter](https://twitter.com/thecubejs)
 
 [![npm version](https://badge.fury.io/js/%40cubejs-backend%2Fserver.svg)](https://badge.fury.io/js/%40cubejs-backend%2Fserver)
 [![GitHub Actions](https://github.com/cube-js/cube.js/workflows/Rust/badge.svg)](https://github.com/cube-js/cube.js/actions?query=workflow%3ARust+branch%3Amaster)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fcube-js%2Fcube.js.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fcube-js%2Fcube.js?ref=badge_shield)
 
-
-Cube Store
-==========
+# Cube Store
```
Comment (ovr) [inline rust/README.md:11]: This file is uploaded to Docker Hub, maybe it's better to introduce a new section inside our documentation?

Comment (hassankhan) [reply rust/README.md:11]: /cc @keydunov

Comment (hassankhan) [reply rust/README.md:11]: /cc @keydunov

Comment (keydunov) [reply rust/README.md:11]: Maybe Docker readme should be a different file? wdyt @hassankhan @ovr

Comment (hassankhan) [reply rust/README.md:11]: Could be, I'm not opposed to it. Might also be good to get @paveltiunov's thoughts too 😄

Comment (keydunov) [reply rust/README.md:11]: Let's just get this one merged for now.

**Pull request merged**